ping/rust/src/bin/testplan_0500.rs

```diff
@@ -1,27 +1,68 @@
 use anyhow::Result;
 use async_trait::async_trait;
 use futures::StreamExt;
-use libp2pv0500::swarm::{keep_alive, NetworkBehaviour, SwarmEvent};
-use libp2pv0500::*;
-use std::collections::HashSet;
-use std::time::Duration;
-use testplan::{run_ping, PingSwarm};
-
-#[async_std::main]
+use libp2pv0500::{
+    core::{
+        either::EitherOutput,
+        muxing::StreamMuxerBox,
+        upgrade::{SelectUpgrade,Version},
+    },
+    identity,
+    mplex,
+    noise,
+    ping,
+    swarm::{
+        keep_alive,
+        NetworkBehaviour,
+        SwarmEvent
+    },
+    tcp,
+    webrtc,
+    yamux,
+    Multiaddr,
+    PeerId,
+    Swarm,
+    Transport,
+};
+use log::{debug, info};
+use rand::thread_rng;
+use std::{collections::HashSet, time::Duration};
+use testplan::*;
+
+#[tokio::main]
 async fn main() -> Result<()> {
     let local_key = identity::Keypair::generate_ed25519();
     let local_peer_id = PeerId::from(local_key.public());
-
-    let swarm = OrphanRuleWorkaround(Swarm::with_async_std_executor(
-        development_transport(local_key).await?,
+    let client = testground::client::Client::new_and_init()
+        .await
+        .expect("Unable to init testground cient.");
```

Comment:
We no longer need the client here now so I'd like to move its creation back into `PingSwarm`.

Reply:
Confirmed this has been removed.